components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsHorizontalBarChart/src/EIAnalyticsHorizontalBarChart.jsx

```diff
@@ -88,6 +88,7 @@ class EIAnalyticsHorizontalBarChart extends Widget {
         this.handlePublisherParameters = this.handlePublisherParameters.bind(this);
         this.handleGraphUpdate = this.handleGraphUpdate.bind(this);
         this.handleStats = this.handleStats.bind(this);
+
```

Comment:
Remove unnessary line

Reply:
Fixed in [ f950da9 ](https://github.com/wso2/analytics-solutions/commit/6f41d5f46cc3f07dc89edbcadc0e34da872c23b8)